packages/react/src/components/MultiSelect/MultiSelect.featureflag.stories.js

```diff
@@ -21,6 +21,29 @@ export default {
       </WithFeatureFlags>
     ),
   ],
+  argTypes: {
+    size: {
+      options: ['sm', 'md', 'lg'],
+      control: { type: 'select' },
+    },
+    direction: {
+      options: ['top', 'bottom'],
+      control: { type: 'radio' },
+    },
+    type: {
+      options: ['inline', 'default'],
+      control: { type: 'radio' },
+    },
+    disabled: {
+      control: { type: 'boolean' },
+    },
+    invalid: {
+      control: { type: 'boolean' },
+    },
```

Comment:
```suggestion
    },
     light: {
      table: {
        disable: true,
      },
    },
```

Same with `light` prop we can remove it here too like the other stories since its deprecated